lib/parameters.py

```diff
@@ -213,6 +213,34 @@ def gen_avc_forced_idr_parameters(spec, profiles):
   params = gen_avc_forced_idr_variants(spec, profiles)
   return keys, params
 
+def gen_avc_tcbrc_idr_variants(spec, profiles):
```

Comment:
TCBRC is a BRC mode, you should refer VBR, but not CQP to generate test params

Reply:
Ok, I have revised and resubmitted